src/wallet/wallet_tx_builder.cpp

```diff
@@ -364,13 +377,19 @@ WalletTxBuilder::IterateLimit(
             // fresh) and once we generate it, hold onto it. But we still don’t have a guarantee
             // that we won’t end up discarding it.
             if (changeAmount > 0 && !changeAddr.has_value()) {
-                changeAddr = GetChangeAddress(
+                auto possibleChangeAddr = GetChangeAddress(
```

Comment:
Nit, non-blocking: I think we use "maybe" as the naming convention here.
```suggestion
                auto maybeChangeAddr = GetChangeAddress(
```